Shadow update

zhichao-ambiq opened the pull request:
feat(shadow): update shadow blur rendering method

- Switched to direct VG-based rendering for shadow blur.
- Updated non-VG rendering method to use GPU-generated rounded corner masks.

zwang-ambiq reviewed (changes requested):
I think we should keep the line 477 to 517 to our private library.

zhichao-ambiq committed:
feat(shadow): update shadow blur rendering method

- Switched to direct VG-based rendering for shadow blur.
- Updated non-VG rendering method to use GPU-generated rounded corner masks.
